### ISSUE TITLE
Links are broken: They are "quoted"

### ISSUE THREAD

**Issue by rugk** [go-gitea/gitea#1477]
- Gitea version (or commit ref): 1.1.0
- Git version: not relevant
- Operating system: Debian Jessie
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
  - [ ] Not relevant
- Log gist: N/A

## Description

Some links to issues e.g. are somehow quoted, so that they destroy the URL:
What happens: `https://domain.example/gittea_subdir/%5C%22/gittea_subdir/organisation/project/issues/13%5C%22` --> `https://domain.exmple/gittea_subdir/\"/gittea_subdir/organisation/project/issues/13\"`
What should happen: `https://domain.example/gittea_subdir/organisation/project/issues/13` (link to issue 13)

Config:
```ini
[server]
ROOT_URL      = https://domain.example/gittea_subdir/
```

It does not affect all URLs however. Only (as far as I found) the ones linking to issues and repos/projects.

Also on the home page, the name of the user is broken:
![grafik](https://cloud.githubusercontent.com/assets/11966684/24839074/5d6423de-1d54-11e7-98f4-f2a4213c39f9.png)

There is also a `"`…

**Comment by cez81** [comment]
Fixed in upcoming 1.1.1, see #1181

**Comment by rugk** [comment]
Okay, so closing as a dupe of https://github.com/go-gitea/gitea/issues/1181.
